editor/package.json

```diff
@@ -28,7 +28,7 @@
     "test-update-snapshot": "pnpm run test -- --updateSnapshot",
     "test-browser-base": "NODE_OPTIONS=$NODE_OPENSSL_OPTION REACT_APP_AUTH0_CLIENT_ID='test' REACT_APP_AUTH0_ENDPOINT='test' REACT_APP_AUTH0_REDIRECT_URI='test' REACT_APP_ENVIRONMENT_CONFIG=development karma start",
     "test-browser": "pnpm run test-browser-base karma-no-shard.conf.js --single-run",
-    "test-browser-debug": "pnpm run test-browser-base karma-no-shard.conf.js --debug",
+    "test-browser-debug": "REACT_APP_BROWSER_TEST_DEBUG='true' pnpm run test-browser-base karma-no-shard.conf.js --debug",
```

Comment:
this is the only unrelated change, it restores the feature that test-browser-debug doesn't mess with the URL so you can reload the browser to re-run the tests